css/style.css

```diff
@@ -0,0 +1,309 @@
+}
+
+table tr:nth-child(even){background-color: #4f4b4b;}
+
+table tr:hover {background-color: rgb(24, 24, 24);}
+
+table, th {
+  padding-top: 12px;
+  padding-bottom: 12px;
+  text-align: left;
+  background-color: rgb(24, 24, 24);
+  color: antiquewhite;
+  border: 5px double;
+  text-align: center;
+  font-size: 20px;
+
+}
+
+
+/* skill section */
+
+#skills {
+    padding: 30px 50px 100px 50px;
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+
+}
+
+.skl {
+    width: 18%;
+    border: 5px double;
+    color: antiquewhite;
+    font-size: 30px;
+    text-align: center;
+    display: inline-block;
+    padding: 16px;
+    margin-bottom: 0.5rem;
+    margin-left: 70px;
+    
+}
+
+/* leave a message section */
+form{
+    width: 25em;
+    max-width: 90%;
+    margin: 0 auto;
+    font-family: 'Blanka', sans-serif;
+    font-size: 15px;
+    color: white;
+    border-radius: 5px;
+    background-color: #222222;
+    padding: 20px;
+
+}  
+     
+   
+input[type=text], select {
+    width: 100%;
+    padding: 12px 20px;
+    margin: 5px 0;
+    display: inline-block;
+    border: 2px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+
+}
+     
+input[type=email], select {
+    width: 100%;
+    padding: 12px 20px;
+    margin: 5px 0;
+    display: inline-block;
+    border: 2px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+
+}
+     
+textarea[type=message] {
+     width: 100%;
+    padding: 20px 20px;
+    margin: 5px 0;
+    border: 2px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+
+}
+     
+button{
+    font-family: 'Blanka', sans-serif;
+    font-size: 15px;
+    background-color: rgb(24, 24, 24);
+    border: none;
+    border-radius: 4px;
+    color: white;
+    padding: 16px 32px;
+    margin: 10px 2px;
+    cursor: pointer;    
+
+}
+   
+  
+/* footer */
+
+.footer-nav{
+    padding-top: 20px;
+    list-style: none;
+    display:block;
+    position: relative;
+   
+}
+
+.container {
+    padding: 1rem 0;
+    margin: 0 auto;
+    max-width: 1024px;
+    width: 90vw;
+    color: antiquewhite;
+    
+}
+
+.fa {
+    padding: 17px;
+    font-size: 30px;
+    width: 50px;
+    text-align: center;
+    text-decoration: none;
+    margin: 5px 5px;
+    border-radius: 70%;
+
+}
+
+.fa-linkedin {
+    background: #007bb5;
+    color: white;
+}
+
+.fa-github {
+    background: #dbd9d9;
+    color: rgb(4, 4, 4);
+}
+
+.fa-twitter {
+    background: #43bcf4;
+    color: rgb(255, 255, 255);
+}
+
+.fa-instagram {
+    background: #f09433; 
+    background: -moz-linear-gradient(45deg, #f09433 0%, #e6683c 25%, #dc2743 50%, #cc2366 75%, #bc1888 100%); 
+    background: -webkit-linear-gradient(45deg, #f09433 0%,#e6683c 25%,#dc2743 50%,#cc2366 75%,#bc1888 100%); 
+    background: linear-gradient(45deg, #f09433 0%,#e6683c 25%,#dc2743 50%,#cc2366 75%,#bc1888 100%); 
+}
+
+
+/* (phones, 600px and down) */
+@media only screen and (max-width: 600px) {
+    .image {
```

Comment:
You have no elements with the class `image` in your HTML so currently these media query styles are not doing anything.